packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -277,7 +280,8 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}> impl
     }
 
     if (FocusTrapZone._focusStack.length && this === FocusTrapZone._focusStack[FocusTrapZone._focusStack.length - 1]) {
-      const focusedElement = document.activeElement as HTMLElement;
+      const doc = getDocument(this._root.current);
+      const focusedElement = doc!.activeElement as HTMLElement;
```

Comment:
I think there are enough uses here that it's worthwhile to make a little helper function and call it directly. Something like:
```tsx
private getDocument() {
  return getDocument(this._root.current);
}
```

Reply:
Agree. But still, make sense to remain in places were `doc` variable is used multiple times in function scope

Reply:
@JasonGore addressed

Reply:
Aren't there more replaceable instances above? I only see once case where it's actually used more than once.

Reply:
I replaced all `document.*` usage with `_getDocument().*`. Only in one place in `FocusZone` and `FocusTrapZone` I saved it to variable `const doc = this._getDocument();` and used as `doc.*` because it is used more than once within the function scope. Let me know if it does make sense to you.